templates/tease-author.twig

```diff
@@ -0,0 +1,27 @@
+<li id="result-row-{{ post.ID }}" class="media search-result-list-item">
+	<img class="d-flex search-result-item-image" src="{{ post.thumbnail.src('thumbnail') }}"
+		 alt="{{ fn('esc_attr', post.thumbnail.alt|default( post.title ) )|raw }}" />
+
+	<div id="tease-{{ post.ID }}" class="media-body search-result-item-body tease tease-{{ post.post_type }}">
+		<div class="search-result-tags">
+			<span class="search-result-item-title">{{ post.get_custom_type }}</span>
+
+			{% for page_type in post.get_custom_terms %}
+				<button class="search-result-item-head no-btn" data-term_id="{{ page_type.term_id }}">{{ page_type.name }}</button>
```

Comment:
there is missing js functionality for this, see [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L51)